src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -111,18 +112,17 @@ private static FileResult generateTextFileResult(FileInfo fileInfo) {
      * {@code fileInfo}.
      */
     private static FileResult generateBinaryFileResult(RepoConfiguration config, FileInfo fileInfo) {
-        String authorsString = GitLog.getBinaryFileAuthors(config, fileInfo.getPath());
-        if (authorsString.isEmpty()) { // Empty string, means no author at all
+        List<String[]> authorsString = GitLog.getFileAuthors(config, fileInfo.getPath());
+        if (authorsString.size() == 0) {
             return null;
         }
 
         Set<Author> authors = new HashSet<>();
         HashMap<Author, Integer> authorContributionMap = new HashMap<>();
 
-        for (String authorString : authorsString.split("\n")) {
-            String[] arr = authorString.split("\t");
-            String authorName = arr[0];
-            String authorEmail = arr[1];
+        for (String[] lineDetails : authorsString) {
+            String authorName = lineDetails[0];
```

Comment:
Is it possible for the author name to be an empty string as well?

Reply:
I've not been able to find any authoritative information on this. The specific documentation of that part is [here](https://git-scm.com/docs/git-config#Documentation/git-config.txt-username). 

I have done a bit of testing, it seems unlike email, setting an empty name does not allow me to commit.
![image](https://user-images.githubusercontent.com/65345505/153789565-7f121336-f185-40ae-969c-197e4579ca4e.png)

Setting it to a full empty string as well prompts the user to set their username. So I believe this should be safe.